Simple 2 argument yes/no version

Gabo6480 opened the pull request:
This version is a simplified version that just uses the <player/viewer> argument and the < permission > argument, returning 'yes' if they have the permission, or 'no' otherwise.

Gabo6480 committed:
Simple 2 argument yes/no version

This version is a simplified version that just uses the <player/viewer> argument and the <permission> argument, returning 'yes' if they have the permission, or 'no' otherwise.

Kqliber reviewed (changes requested):
The yes/no returns should be changed to use `PlaceholderAPIPlugin.booleanTrue()` and `PlaceholderAPIPlugin.booleanFalse()` so that users can define their desired outputs using the existing config options

Gabo6480 committed:
Update RelConExpansion.java

Instead of static 'yes' 'no' strings as a result, it now uses PlaceholderAPIPlugin.booleanTrue() and PlaceholderAPIPlugin.booleanFalse().
